### ISSUE TITLE
Add content for "How do we CD with a mud ball?"

### ISSUE THREAD

**Issue by bdfinst** [Minimum-CD/cd-manifesto#200]
### Discussed in https://github.com/Minimum-CD/cd-manifesto/discussions/199

<div type='discussions-op-text'>

<sup>Originally posted by **rlabercrombie** April 15, 2022</sup>
Given the latest presentation at devnexus (great job by the way!) there were a few questions about 'how to get there' with tough architectures; in this example they mentioned having to have a many year long process of transitioning from VMs to x/y/z prior to being able to achieve (or it sounded like even start) minimal cd.  

Would there be any interest in creating a new section or adding to existing sections for use-cases fitting the bill of 'getting to a minimum viable cd system: best practices (or examples... whichever would fit best)?  

As an example of a use-case: You have a monolith with zero testing coverage.   Separating out a small chunk of code - ideally on an appropriate domain/business seam - into its own project, adding some unit (or other) tests and getting a basic pipeline working as a way of an iterative process? Then you could have parallel work on improving that pipeline and pulling off another small chunk of code.  

Or is this out of scope?</div>

**Comment by Jerreck** [comment]
i think this would be a great thing to have experience reports for

**Comment by ibathazi** [comment]
Creating a high level recommendation path to dig out from the hole many teams are in, I think that could help many .

**Comment by bdfinst** [comment]
I'm going to write an article about this, because I get this question constantly, and I'll link it over here.

**Comment by esmiralha** [comment]
Any updates on that article? :)

**Comment by tracybannon** [comment]
I agree that this is an experience report as well as high level recommendations for addressing this.  It is HIGHLY prevalent in my world.  @bdfinst  - Glad to coauthor or be a feedback provider.
